Cesium3DTiles/include/Cesium3DTiles/TileContentLoadInput.h

```diff
@@ -6,6 +6,8 @@
 #include "Cesium3DTiles/TileContext.h"
 #include "Cesium3DTiles/TileID.h"
 #include "Cesium3DTiles/TileRefine.h"
+#include "Cesium3DTiles/Tileset.h"
```

Comment:
I thinks it's better that TileContentLoadInput should require TilesetContentOption in its constructor, so that it doesn't depend on Tileset class.